### ISSUE TITLE
[BITV] 9.1.4.3/3.7 - The labeling of non-selected menu items in the main navigation in the upper left corner of the page does not meet the text contrast requirements when navigation is hovered with the mouse or when focused by keyboard. (4)

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/server#36957]
- [ ] Fix
- [ ] Backport

![d0195334059bd2d75916982362f0bdbd](https://user-images.githubusercontent.com/1315170/222224605-368ccf62-03a8-4051-87fa-4fc0e7ce5792.png)

<details>
<summary>Details</summary>

https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6fe46b78da-v3-n7

</details>

- [ ] 9.1.4.3/6.3 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6fe46b78da-v6-n3
- [ ] 9.1.4.3/9.4 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6fe46b78da-v9-n4
- [ ] 9.1.4.3/10.7 - https://report.bitvtest.de/default-en/d63601ac-cb34-4645-8256-66bec78964a0.html#checkpoint-6fe46b78da-v10-n7

**Comment by Fenn-CS** [comment]
This now appears fixed?

![navbar-contrast](https://github.com/nextcloud/server/assets/14317775/bf7eb290-91db-4a7a-a68b-e175f7f03256)


cc: @JuliaKirschenheuter

**Comment by JuliaKirschenheuter** [comment]
@Fenn-CS yes, this one is already fixed 🥳

**Comment by JuliaKirschenheuter** [comment]
fixed by https://github.com/nextcloud/server/pull/38519
